addons/web/static/src/views/list/list_renderer.js

```diff
@@ -787,7 +787,7 @@ export class ListRenderer extends Component {
 
     onButtonCellClicked(record, column, ev) {
         if (!ev.target.closest("button")) {
-            this.onCellClicked(record, column);
+            this.onCellClicked(record, column, ev);
```

Comment:
@ged-odoo, @aab-odoo are you ok with this? I guess a test is not required for this, right?

Reply:
@mba-odoo is it solved without your fix now?

Reply:
@IT-Ideas  yes it is solved. Thanks.